<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderAndCheckWidget</class>
 <widget class="QWidget" name="OrderAndCheckWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="CheckOrderWidget" name="checkOrderPage"/>
     <widget class="OrderGoodsWidget" name="orderGoodsPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrderGoodsWidget</class>
   <extends>QWidget</extends>
   <header>OrderGoodsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckOrderWidget</class>
   <extends>QWidget</extends>
   <header>CheckOrderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
